notices/tests.py

```diff
@@ -1,3 +1,50 @@
 from django.test import TestCase
-
+from .models import Notice,NoticeBranchYear,BookmarkedNotice
+from profiles.models import FacultyDetail
+from django.contrib.auth.models import User
 # Create your tests here.
+
+#models test
+class NoticeTest(TestCase):
+    
+    def setUp(self):
+        user=User.objects.create(username='Ajay',password='5678')
```

Comment:
@insomniac12 can you change `user` with `self.user` and similar for the `faculty` and `notice` at line number 12 and 13.